client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -1,47 +1,102 @@
 /**
  * External dependencies
  */
-const React = require( 'react' );
+import React, { Component, PropTypes } from 'react';
+import debug from 'debug';
 
 /**
  * Internal dependencies
  */
-const loadScript = require( 'lib/load-script' ).loadScript;
+import loadScript from 'lib/load-script';
 
-module.exports = React.createClass( {
-	displayName: 'EditorMediaModalDetailPreviewVideoPress',
+/**
+ * Module variables
+ */
+const log = debug( 'calypso:post-editor:videopress' );
+const videoPressUrl = 'https://wordpress.com/wp-content/plugins/video/assets/js/next/videopress.js';
+
+class EditorMediaModalDetailPreviewVideoPress extends Component {
+	static propTypes = {
+		isPlaying: PropTypes.bool,
+		item: PropTypes.object.isRequired,
+	};
+
+	static defaultProps = {
+		isPlaying: false,
+	};
 
-	propTypes: {
-		item: React.PropTypes.object.isRequired
-	},
+	constructor() {
+		super();
+
+		this.onScriptLoaded = this.onScriptLoaded.bind( this );
```

Comment:
Optional, but you can avoid constructor by defining the function as a class instance initializer

```js
onScriptLoaded = () => {
	// ...
};
```

https://tc39.github.io/proposal-class-public-fields/